.gitignore

```diff
@@ -109,3 +109,6 @@ lib
 old_src
 
 build.sh
+
+# lock file
+yarn.lock
```

Comment:
Actually, we need yarn.lock file to be present to be able to cache him for github workflow.

Reply:
I see, i will get the yarn.lock back.